python/pacemaker/Makefile.am

```diff
@@ -7,6 +7,7 @@
 # or later (GPLv2+) WITHOUT ANY WARRANTY.
 #
 
+CLEANFILES = buildoptions.py
 MAINTAINERCLEANFILES    = Makefile.in
 
 pkgpython_PYTHON = __init__.py \
```

Comment:
Doesn't buildoptions.py need to be in pkgpython_PYTHON instead of CLEANFILES, to get installed?